tests/test_integration/test_run_transfer/test_hdfs.py

```diff
@@ -174,7 +185,7 @@ async def test_run_transfer_hdfs_to_postgres(
     df = reader.run()
 
     # as Excel does not support datetime values with precision greater than milliseconds
```

Comment:
ORC and Parquet should support microseconds...

Reply:
Yes, and it works for Postgres -> HDFS, but not vice versa. Perhaps smth happens in fixture, looking